src/module_PI/Raspberry_PI/main/DataPacket.java

```diff
@@ -60,6 +65,12 @@ private void parseLogLevelFromByteArray(){
 		packetLogLevel = PacketLogLevel.getLogLevelFromByte(packetCodeByteArray[LOG_LEVEL_INDEX]);
 	}
 	
+	private void parseMessage(){
+		int packetIntModThisIsMessageInt = (int) Math.pow(10, NUM_OF_MESSAGE_BYTES_IN_PACKET);
```

Comment:
This name was confusing to try and figure out what it was. I had to read over it a few times and look at the code below to figure it out. Possibly breaking from convention here and naming it something like `packetInt_modThisInt_isMessageInt` might make it a little more clear to an outsider looking at this code. 

Its a fairly complex idea you are trying to capture in the name of this variable name. Maybe take a step back and name what it _is_ instead of _what it will be used for_.